packages/protocol/test/common/accounts.ts

```diff
@@ -0,0 +1,330 @@
+import { upperFirst } from 'lodash'
+import { AccountsInstance } from 'types'
+import { getParsedSignatureOfAddress } from '../../lib/signing-utils'
+import {
+  assertLogMatches,
+  assertLogMatches2,
+  assertRevert,
+  NULL_ADDRESS,
+} from '../../lib/test-utils'
+const Accounts: Truffle.Contract<AccountsInstance> = artifacts.require('Accounts')
+let authorizationTests: any = {
```

Comment:
How is this used? It looks like it's overwritten immediately?

Reply:
So this is tricky as at start up time, we need these values in here to properly construct the test descriptions. Once the test runs, this object indeed will get overwritten with the runtime values such as references to the contract methods. I'll separate out both concerns to reduce confusion